### DISCUSSION TITLE
win7 32byte cannot run customised app

### DISCUSSION THREAD

**Discussion by zhangweimtan** [tw93/Pake#318] [Q&A]
cargo.toml
changed as :
    [dependencies]
    serde_json = "1.0.91"
    serde = { version = "1.0.152", features = ["derive"] }
    **_tauri = { version = "1.2.4", features = ["windows7-compat"] }_**
    image = "0.24.5"
    home = "0.5.4"
    tauri-utils = "1.2.1"
    webbrowser = "0.8.7"
    wry = "0.23.4"
    dirs = "4.0"

npm run tauri build -- --target i686-pc-windows-msvc

win7 has not installed webview2, and it is downloaded and  installed.

the app shown on the desktop but could not run the web.

**Comment by Tlntin** [comment]
i suggest you can ask to [tauri](https://github.com/tauri-apps/tauri).
